### ISSUE TITLE
openindiana VM broken

### ISSUE THREAD

**Issue by ThomasWaldmann** [borgbackup/borg#3708]
```
    openindiana64: /tmp/vagrant-shell: line 2: cd: /vagrant/borg: No such file or directory
    openindiana64: Already using interpreter /usr/bin/python3
    openindiana64: Using base prefix '/usr'
    openindiana64: New python executable in /export/home/vagrant/borg-env/bin/python3
    openindiana64: Also creating executable in /export/home/vagrant/borg-env/bin/python
    openindiana64: Installing setuptools, pip, wheel...
    openindiana64: done.
==> openindiana64: Running provisioner: install borg (shell)...
    openindiana64: Running: inline script
    openindiana64: /tmp/vagrant-shell: line 2: cd: /vagrant/borg: No such file or directory
    openindiana64: Requirement already up-to-date: wheel in ./borg-env/lib/python3.4/site-packages
    openindiana64: /tmp/vagrant-shell: line 5: cd: borg: No such file or directory
    openindiana64: find: stat() error src: No such file or directory
    openindiana64: Could not open requirements file: [Errno 2] No such file or directory: 'requirements.d/development.txt'
    openindiana64: python: can't open file 'setup.py': [Errno 2] No such file or directory
    openindiana64: Directory '.' is not installable. File 'setup.py' not found.
```

**Comment by ThomasWaldmann** [comment]
https://github.com/hashicorp/vagrant/issues/9614

**Comment by ThomasWaldmann** [comment]
vagrant 2.1.1: no changelog entry about this, ticket still open, so likely still broken.

**Comment by ThomasWaldmann** [comment]
retry with vagrant 2.1.3 (fixing the issue has been added to vagrant 2.1.3 milestone).

**Comment by ThomasWaldmann** [comment]
i manually applied the fix made for 2.1.3 and it worked:
```
    openindiana64:   py34: commands succeeded
```
